src/main/java/network/brightspots/rcv/CastVoteRecord.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Universal RCV Tabulator
+ * RCTab
  * Copyright (c) 2017-2020 Bright Spots Developers.
```

Comment:
Can you please update the copyright date to 2022 for all these files while you're at it?

Reply:
I think it's better to do this along with the license updates in a different PR.

Reply:
I fixed the copyright template, and updated the comments in Main.java.  The .rtf here is a document we received from Clear Ballot, so not something we want to touch.